schemas/donor.json

```diff
@@ -0,0 +1,59 @@
+{
+  "description": "The collection of data elements related to a specific donor in a specific ICGC-ARGO program.",
+  "fields": [
+    {
+      "description": "Unique identifier of the ARGO program.",
+      "name": "program_id",
+      "restrictions": {
+        "required": true
+      },
+      "valueType": "string"
+    },
+    {
+      "description": "Unique identifier of the donor, assigned by the data provider.",
+      "name": "submitter_donor_id",
+      "restrictions": {
+        "required": true
```

Comment:
should all `submitter_*_id` have regex pattern? at least to avoid special characters (eg, `/`, `*`, `?` etc) being used